packages/pages/src/components/hoursStatus/hoursStatus.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useEffect, useState } from "react";
+import c from "classnames";
+import { HoursType } from "../hours/types.js";
+import {
+  HoursIntervalManipulator,
+  HoursManipulator,
+} from "../hours/hoursManipulator.js";
+
+export interface StatusParams {
+  isOpen: boolean;
+  currentInterval: HoursIntervalManipulator | null;
+  futureInterval: HoursIntervalManipulator | null;
+  timeOptions?: Intl.DateTimeFormatOptions;
+  dayOptions?: Intl.DateTimeFormatOptions;
+}
+
+export interface TemplateParams {
+  currentTemplate?: (s: StatusParams) => React.ReactNode;
```

Comment:
I'm not sure how others feel but all of these props being call fooTemplate sounds weird to me.